### PR TITLE
Code formatting

### PR DESCRIPTION
**Pull request by OneSecure**
To remove warnings in Xcode. Like the image.
> https://pbs.twimg.com/media/C41h4KiVcAArb79.jpg

### CODE REVIEW

File: FHSTwitterEngine/FHSTwitterEngine.h [hunk 1]
```diff
@@ -619,15 +618,15 @@ extern NSString * const FHSErrorDomain;
 /**
  Add users to a list.
  @param listID List id.
- @param List of users.
+ @param users of users.
```
Comment (ReadmeCritic) [inline FHSTwitterEngine/FHSTwitterEngine.h]: This should say

`@param users List of users`

File: FHSTwitterEngine/FHSTwitterEngine.h [hunk 2]
```diff
@@ -619,15 +618,15 @@ extern NSString * const FHSErrorDomain;
...
  @return If an error occurs, returns an NSError object that describes the problem.
  */
 - (NSError *)addUsersToListWithID:(NSString *)listID users:(NSArray *)users;
 
 /**
  Remove users from a list.
  @param listID List id.
- @param List of users.
+ @param users of users.
```
Comment (ReadmeCritic) [inline FHSTwitterEngine/FHSTwitterEngine.h]: same as above

**Review by ReadmeCritic — CHANGES_REQUESTED** [review]

**Comment by dkhamsing** [comment]
@OneSecure thanks for the pull request, can you look at the comments :)

**Comment by OneSecure** [comment]
Following your words, changed.

**Review by dkhamsing — APPROVED** [review]

**Comment by dkhamsing** [comment]
Published with `2.0.6` 🎉

**Pull request merged**